Library/Formula/boot.rb

```diff
@@ -0,0 +1,18 @@
+require "formula"
+
+class Boot < Formula
+  homepage "http://boot-clj.com"
+  version "2.0.0-rc4"
```

Comment:
@mikemcquaid The Jar file is inlined within the shell script and directly piped into `java`. The shell script is 7mb. When building from source Maven would be required but for usage Maven isn't required.